site/public/js/gradeable-notebook.js

```diff
@@ -194,15 +190,26 @@ $(document).ready(() => {
             $(clear_button_id).attr('disabled', true);
         }
         else {
-            $(clear_button_id).attr('disabled', false);
+            const value = $(clear_button_id).attr('data-older_version');
+            if (value) {
+                $(clear_button_id).attr('disabled', true);
+            }
+            else {
+                $(clear_button_id).attr('disabled', false);
+            }
```

Comment:
Can condense by coercing `value` to a boolean using `!!` notation:
```suggestion
            $(clear_button_id).attr('disabled', !!$(clear_button_id).attr('data-older_version'));
```